src/main/java/org/folio/qm/validation/impl/common/OnlyOne100ControlFieldValidationRule.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.qm.validation.impl.common;
+
+import org.folio.qm.domain.dto.FieldItem;
+import org.folio.qm.domain.dto.MarcFormat;
+import org.folio.qm.validation.FieldValidationRule;
+import org.folio.qm.validation.ValidationError;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.folio.qm.converter.elements.Constants.TAG_001_CONTROL_FIELD;
+import static org.folio.qm.domain.dto.MarcFormat.AUTHORITY;
+import static org.folio.qm.domain.dto.MarcFormat.BIBLIOGRAPHIC;
+import static org.folio.qm.domain.dto.MarcFormat.HOLDINGS;
+
+@Component
+public class OnlyOne100ControlFieldValidationRule extends FieldValidationRule {
+  @Override
+  protected Optional<ValidationError> validate(List<FieldItem> fieldItems) {
+    var fieldsWith008TagCode = filterFieldsByTagCode(fieldItems, TAG_001_CONTROL_FIELD);
+    return onlyOneRequiredCondition().apply(TAG_001_CONTROL_FIELD, fieldsWith008TagCode);
```

Comment:
Actually having a 001 tag is not required. For example, when creating new holdings record UI doesn't send a 001 tag in a request. I suggest creating a new condition that will check that there are no greater than one 001 tags existing.